### DISCUSSION TITLE
Ability to search for a setting

### DISCUSSION THREAD

**Discussion by assayire** [labithiotis/zed-themes#130] [Ideas]
A search bar in the left pane to search for a setting would be immensely useful

**Comment by labithiotis** [comment]
I like this, but how do you envision that working? Should it filter to show only matching keys/input or scroll to match?

Another option is to combine with the feature for highlighting/scrolling to each key when hovering over elements in preview and also have a search that works similarly.

**Comment by assayire** [comment]
I would go with whichever is easier to implement.

One way is to filter only matching element by search text.

... and yes if when searching the search text matches an element exactly by name, you could select the item on the left pane and highlight the UI element on the right.
